hw3/css/stylehw3.css

```diff
@@ -0,0 +1,477 @@
+* {
+	box-sizing: border-box;
+}
+body{
+	font: 14px/26px Arial, sans-serif;
+}
+/*HEADER*/
+.header {
+	background-image: url(../img/header.png);
+	background-repeat: repeat-x;
+	box-shadow: 0px 1px 13px 0px rgba(0, 0, 0, 0.294);
+	position: relative;
+}
+.header__wrapper {
+	width: 80%;
```

Comment:
you can move this two styles to separate class and use it for each wrapper block